pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -764,6 +764,14 @@ func (m *kubeGenericRuntimeManager) SyncPod(ctx context.Context, pod *v1.Pod, po
 		// When runc supports slash as sysctl separator, this function can no longer be used.
 		sysctl.ConvertPodSysctlsVariableToDotsSeparator(pod.Spec.SecurityContext)
 
+		// Prepare resources for pod containers
+		for _, container := range append(pod.Spec.InitContainers[:], pod.Spec.Containers...) {
+			err := m.runtimeHelper.PrepareResources(pod, &container)
```

Comment:
Without tracing through the code right now -- remind me -- does this ultimately just end up calling `Kubelet.PrepareResources()`?

Reply:
yes, it's calling `klet.PrepareResources`, which calls `containerManager.PrfepareResources`, which calls `draManager.PrepareResources`.